dev-util/poke/poke-9999.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit elisp-common flag-o-matic
+
+DESCRIPTION="Extensible editor for structured binary data"
+HOMEPAGE="https://www.jemarch.net/poke"
+
+if [ "${PV}" = 9999 ]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.savannah.gnu.org/git/poke.git"
+else
+	SRC_URI="mirror://gnu/poke/${P}.tar.gz"
+	KEYWORDS=""
```

Comment:
Why is this blank?

Reply:
to fill in when bumping, but I could just add ~amd64 there